src/app/frontend/common/components/uploadfile/component.ts

```diff
@@ -44,6 +47,17 @@ export class UploadFileComponent {
       } as KdFile);
     };
 
-    reader.readAsText(file);
+    if (file instanceof ArrayBuffer) {
+      this.reportError(
+          'File Format Error',
+          'Argument of type string | ArrayBuffer is not assignable to parameter of type string');
```

Comment:
This is like an error for a developer not for the user. It should be simpler. What about `Specified file has the wrong format`?

Reply:
ok changing that and is the title `File Format Error` good?